nss-day-cohort-29/nutshell-spirit-wind
# ab-loginwork

# Description

For this pull, I've added functions to the Login and Register User buttons to hide the appropriate elements. Fixed an issue where alert looped through all instances of users for invalid login. When a user successfully logs in, the dashboard appears and login elements are hidden.


## Type of change





# Testing Instructions for Change Made

1. git fetch --all
2. git checkout ab-loginwork
3. go to src/lib
4. grunt
5. Click Register button and make sure Login elements disappear
6. Register a new user and make sure returned to login elements automatically after register
7. Log in with newly created user and see if "Dashboard" appears
8. Try log in with invalid credentials to see alert message.


# Checklist:
